### ISSUE TITLE
[Feature Request] Prompt for bw login and export BW_SESSION

### ISSUE THREAD

**Issue by conor-byrne** [andykais/bitwarden-dmenu#7]
I'm not a JS dev, otherwise I'd love to try to implement this myself.
A way to login though dmenu (if not already logged in) would be amazing.

The bash command corresponding to the required action: `export BW_SESSION="$(bw login --raw)"`
The fish command for those who use fish (me): `set BW_SESSION (bw login --raw)`

**Comment by andykais** [comment]
Hi there! This is something I plan to add soon, it is listed on the Projects page as one of my todos https://github.com/andykais/bitwarden-dmenu/projects/1#card-13432787. 

To make this happen though, I have to wait for https://github.com/bitwarden/cli/issues/27 to complete before I can implement it. Otherwise, I have no reliable way of knowing whether the user is logged in already or not

**Comment by andykais** [comment]
The upstream issue above has been resolved! (merged in https://github.com/bitwarden/cli/pull/47) I will begin working on this feature as soon as it is published to npm.

**Comment by conor-byrne** [comment]
Hey @andykais, just giving you an update, when I enter my password I am given the following error:

```
SyntaxError: Unexpected token Y in JSON at position 0
    at JSON.parse (<anonymous>)
    at getAccounts (/usr/lib/node_modules/bitwarden-dmenu/src/index.js:53:21)
    at module.exports (/usr/lib/node_modules/bitwarden-dmenu/src/index.js:107:16)
    at <anonymous>
bitwarden is locked.
clipboard is cleared.
```

Let me know if there's anything else I can do from a troubleshooting end :smile:

**Comment by andykais** [comment]
Hey there @conor-byrne. You'll have to forgive me, sometimes life gets in the way of open source. Anyways, I have released [v1.5.0](https://github.com/andykais/bitwarden-dmenu/compare/85a1466d58034d74e1e9e6248be89e77124f4f76...c8c6cdfeaec46b79ba559d7e83555c21483b7331)! I am beginning better version management (starting with using github releases). 

So to answer the initial feature request, `bitwarden-dmenu` now supports login flow. 

To address the bug report attached I have improved the bug report workflow.
I have a new [bug report issue template](https://github.com/andykais/bitwarden-dmenu/issues/new?template=bug_report.md) which describes what you can provide to make troubleshooting easier. I am going to close this issue as I have addressed the initial feature request. If you still are experiencing the same bug, please open a new issue!
